plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1083,10 +1097,10 @@ def main():
         try:
             vm = get_vm(proxmox, vmid)
             if not vm:
-                module.exit_json(changed=False, msg="VM %s does not exist" % vmid)
+                module.exit_json(changed=False)
 
             if getattr(proxmox.nodes(vm[0]['node']), VZ_TYPE)(vmid).status.current.get()['status'] == 'running':
-                module.exit_json(changed=False, msg="VM %s is running. Stop it before deletion." % vmid)
+                module.exit_json(changed=False, msg="VM %s is running. Stop it before deletion or use force=yes." % vmid)
```

Comment:
This change does not belong in this PR, but in the other.

Reply:
You are right I forgot to remove this part.

Reply:
Corrected.